plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -6,54 +6,60 @@ import Breadcrumb from "#SRC/js/components/Breadcrumb";
 import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
 import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
 
-export default function Breadcrumbs({ item, children, renderStates }) {
-  function getBreadcrumb(item, id = "", name = "") {
-    const isDetailPage = [...item.path, item.name].join(".") === id;
-    const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
-
+export default function Breadcrumbs({ path, name, children, extra }) {
+  /**
+   * @param  {string} link
+   * @param  {string[]} pathname path including name (if given)
+   * @param  {JSX.Element} [extra=null] extra JSX Components to render
+   *
+   * @return {JSX.Element} Breadcrumb Part Component
+   */
+  function buildJSX(link, pathname, extra = null) {
```

Comment:
I think the function name has little to no meaning as does the extra.

the variable name pathname reads like a string, can we make it paths to be clear that it means an array?

Reply:
re fname: agree, do you have a better idea?

re pathname: I went with pathname because it contains path + name in some situations. 🤔 also its typed array, so it should be obvious 🤔 do you have a better idea?